build/run

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# Mount the project as a read only filesystem to /app and pass
+# along any arugments that were provided to this script
+docker run -v"$(pwd):/app:ro" backus/rubocop-rspec "$@"
```

Comment:
`--read-only` does not work! That makes the entire FS read only compared to just the volume.